### ISSUE TITLE
False positive when excluding transitive dependency

### ISSUE THREAD

**Issue by hkupty** [nebula-plugins/gradle-lint-plugin#366]
Hi, I'm using nebula-plugins to help me migrate from maven to gradle and it really did help figuring out either missing declared dependencies or unnecessary dependencies. However, it is giving me a few false-positives. From what I can see, those happen when the dependency is declared w/ an interpolated version number. The most obvious case is this one where I'm excluding a transitive dependency [that is explicitly declared in said package's pom.xml](https://github.com/Litote/kmongo/blob/master/kmongo-flapdoodle/pom.xml#L50-L53) but shows up as an unused exclude.

```
warning   unused-exclude-by-dep              the excluded dependency is not a transitive of org.litote.kmongo:kmongo-flapdoodle:$kmongoVersion, so has no effect. See https://github.com/nebula-plugins/gradle-lint-plugin/wiki/Unused-Exclude-Rule for more details
libs/kotlin/test-utils/build.gradle:20
exclude group: 'org.slf4j', module: 'slf4j-simple'
```

**Comment by boris-petrov** [comment]
I believe this is a duplicate of #224.
